{
  try {
    return method.invoke(proxy,args);
  }
 catch (  InvocationTargetException ex) {
    throw ex.getCause();
  }
}
